### ISSUE TITLE
Don't force me to pick a default editor during installation

### ISSUE THREAD

**Issue by thomaslevesque** [git-for-windows/git#2439]
### Setup

 - Which version of Git for Windows are you using? Is it 32-bit or 64-bit?
```
git version 2.24.0.windows.1
cpu: x86_64
built from commit: 6a3fa2a1833f3c7ac71cefefe44b3a4b4138e220
sizeof-long: 4
sizeof-size_t: 8
```

 - Which version of Windows are you running? Vista, 7, 8, 10? Is it 32-bit or 64-bit?

```
Microsoft Windows [Version 10.0.18362.535]
```

 - What options did you set as part of the installation? Or did you choose the
   defaults?

N/A (the problem is during the installation, before it's complete)

 - Any other interesting things about your environment that might be related
   to the issue you're seeing?

N/A

### Details

 - Which terminal/shell are you running Git from?

Powershell

 - What commands did you run to trigger this issue? If you can provide a
   [Minimal, Complete, and Verifiable example](http://stackoverflow.com/help/mcve)
   this will help us understand the issue.

Run installer to update Git

 - What did you expect to occur after running these commands?

Keep my current editor settings (which is not in the list proposed by the installer)

 - What actually happened instead?

I'm required to pick one from the list, or select a custom one. But I just want to keep what I already have.

 - If the problem was occurring with a specific repository, can you provide the
   URL to that repository to help us with testing?

N/A

**Comment by thomaslevesque** [comment]
Also, it's really annoying that the installer prompts me for every setting. When I upgrade, I don't want to change my settings, I want to keep them. With the current experience, I have to think about which value I want for each setting. It makes the upgrade process really annoying, and discourages from doing frequent upgrades.

**Comment by rimrul** [comment]
This should be solved by git-for-windows/build-extra#270. Please check wether the next installer keeps your editor (and other settings) as expected.

**Comment by thomaslevesque** [comment]
@rimrul thanks!
I'll close this issue, since it's basically the same as the other one.
